src/runtime/HalideBuffer.h

```diff
@@ -349,6 +369,49 @@ class Buffer {
         return false;
     }
 
+    /** Crop a single dimension without handling device allocation. */
+    void crop_host_side(int d, int min, int extent) {
+        // assert(dim(d).min() <= min);
```

Comment:
Are these meant to be commented out?

Reply:
They're copied from the original code. Will try to figure out what is up as soon as I figure out why it is failing tests on some platforms.